adapters/adnuntius/adnuntius.go

```diff
@@ -193,19 +197,10 @@ func (a *adapter) generateRequests(ortbRequest openrtb2.BidRequest) ([]*adapters
 }
 
 func (a *adapter) MakeBids(request *openrtb2.BidRequest, externalRequest *adapters.RequestData, response *adapters.ResponseData) (*adapters.BidderResponse, []error) {
-	if response.StatusCode == http.StatusNoContent {
-		return nil, nil
-	}
 
 	if response.StatusCode == http.StatusBadRequest {
 		return nil, []error{&errortypes.BadInput{
-			Message: fmt.Sprintf("Unknown status code: %d.", response.StatusCode),
-		}}
-	}
-
-	if response.StatusCode != http.StatusOK {
```

Comment:
I suggest keeping an error code catch-all in case something goes wrong on your server so PBS can handle it gracefully.